### PR TITLE
catch up with all changes

### PR DESCRIPTION
**Pull request by ZeratuuLL**
fixed last night's issue

### CODE REVIEW

File: hdt_util/hdt_util/evaluation.py [hunk 1]
```diff
@@ -203,25 +213,60 @@ def evaluate_model(self, model_args, geo_type='state', geo_values=None, data_sou
...
-        if real_as_of_dates[0] > today:
-            print('No enough data to evaluate the model! You have to wait until {} for evaluation data to be available'.format(real_as_of_dates[0]))
-            return None, None
-        
+        delay_dist = model_args['delay_dist']
         model = ARLIC(**model_args)
+        optimizer = tf.keras.optimizers.Adam(lr=0.1)
+        loss = tf.keras.losses.MSE
+        
+        def scheduler(epoch, lr):
```
Comment (sxsheng) [inline hdt_util/hdt_util/evaluation.py:232]: We can remove `scheduler `and `scheduler_callback`.

File: hdt_util/hdt_util/evaluation.py [hunk 2]
```diff
@@ -203,25 +213,60 @@ def evaluate_model(self, model_args, geo_type='state', geo_values=None, data_sou
...
-        real_prediction_date = self.end_date + datetime.timedelta(days=self.max_prediction_length) # dates that we have to make a prediction
-        real_as_of_date = real_prediction_date + datetime.timedelta(days=self.delay) # considering delay, what are the as_of_date we need to evaluation each prediction
-        today = datetime.date.today()
-        if real_as_of_dates[0] > today:
-            print('No enough data to evaluate the model! You have to wait until {} for evaluation data to be available'.format(real_as_of_dates[0]))
-            return None, None
-        
+        delay_dist = model_args['delay_dist']
         model = ARLIC(**model_args)
+        optimizer = tf.keras.optimizers.Adam(lr=0.1)
```
Comment (sxsheng) [inline hdt_util/hdt_util/evaluation.py:229]: Also change the learning rate to 1

File: hdt_util/hdt_util/evaluation.py [hunk 3]
```diff
@@ -258,39 +299,71 @@ def evaluate_model(self, model_args, geo_type='state', geo_values=None, data_sou
...
+            li.value = tf.clip_by_value((cases_std*(li.value-li_mean)/li_std) + cases_mean, clip_value_min=0,clip_value_max=float('inf'))       
+
+            temp_li = Delay.deconv(li, delay_dist)
+            temp_li = tf.reshape(temp_li, shape=(1,-1,1))
             
+            #fit model and reload best weights
+            args = {
+                "x":temp_li,
+                "y":temp_cases,
+                "epochs":1000,
```
Comment (sxsheng) [inline hdt_util/hdt_util/evaluation.py:336]: change epochs to 10,000

**Review by sxsheng — CHANGES_REQUESTED** [review]
Just some minor changes to ARLIC_evaluator

File: hdt_util/hdt_util/evaluation.py [hunk 4]
```diff
@@ -203,25 +213,60 @@ def evaluate_model(self, model_args, geo_type='state', geo_values=None, data_sou
...
+            filepath=model_path,
+            verbose=0,
+            monitor="mean_squared_error",
+            save_best_only=True,
+            save_weights_only=True
+        )
+        
+        callbacks = [
+            checkpoint_callback,
+            scheduler_callback,
```
Comment (sxsheng) [inline hdt_util/hdt_util/evaluation.py:261]: remove scheduler_callback

**Review by sxsheng — APPROVED** [review]
